Jenkinsfile

```diff
@@ -26,7 +26,7 @@ pipeline {
         stage ('Static analysis') {
             steps {
                 sh "pylint ./${KATPACKAGE} --output-format=parseable --exit-zero > pylint.out"
-                sh "lint_diff.sh -r ${KATPACKAGE}"
+                //sh "lint_diff.sh -r ${KATPACKAGE}"
```

Comment:
@amakhaba 
Not entirely happy about but if we are doing it such that we can get the PR in and proceed to the autoformat stage then I am happy.

Reply:
Yeah, the idea is to uncomment it after the code-autoformat stage.

Reply:
thanks and great!